Code/Framework/AzToolsFramework/AzToolsFramework/ViewportUi/ButtonGroup.cpp

```diff
@@ -70,6 +79,16 @@ namespace AzToolsFramework::ViewportUi::Internal
         return buttonId;
     }
 
+    bool ButtonGroup::RemoveButton(ButtonId buttonId)
+    {
+        if (m_buttons.find(buttonId) != m_buttons.end())
+        {
+            m_buttons.erase(buttonId);
+            return true;
+        }
+        return false;
+    }
```

Comment:
There is no need to check if the buttonId is in the map, just [erase it](https://en.cppreference.com/w/cpp/container/unordered_map/erase).
```suggestion
        return m_buttons.erase(buttonId) != 0;
```